evap/evaluation/migrations/0132_add_infotext_model.py

```diff
@@ -0,0 +1,113 @@
+from django.db import migrations, models
+from evap.evaluation.models import Infotext
+
+
+def create_infotexts(apps, _schema_editor):
+    infotext = apps.get_model("evaluation", "Infotext")
+
+    infotexts = [
+        (
+            "Information for contributors",
```

Comment:
I removed it, and now the tests cannot find the infotexts :cry: . If I visit the pages manually, they still show up (even after a clean `reload_testdata`)
But in general, what about production? The infotexts need to be in the "global state" of EvaP, so I'd say they should be kept in the migrations but removed in the test data. I'm not sure if I understood it correctly though

Reply:
The idea is to have the texts within the test data so that we have some plausible texts when testing or using the page during development. To avoid duplicated data, their contents don't need to be added by a migration (but they need to be created empty nevertheless). The initial content on production will be changed manually (we do it like this for all data of this kind, e.g., FAQ texts or email templates).